sources/views/form.js

```diff
@@ -1,21 +1,79 @@
 import { JetView } from "webix-jet";
+import { contacts } from "models/contacts";
+import { countries } from "models/countries";
+import { statuses } from "models/statuses";
+
 
 export default class Form extends JetView {
 	config() {
+		const _ = this.app.getService("locale")._;
 		return {
 			paddingX: 5,
 			view: "form",
+			width: 400,
 			localId: "form",
 			elements: [
 				{
 					rows: [
-						{ view: "text", label: "User Name", name: "title" },
-						{ view: "text", label: "Email", name: "email" }
-
-					]
+						{ view: "text", label: _("Name"), name: "Name", invalidMessage: "Name can not be empty" },
+						{ view: "text", label: _("Email"), name: "Email", invalidMessage: "Email can not be empty or wrong format" },
+						{ view: "combo", label: _("Country"), value: 1, name: "Country", options: { body: {template:"#Name#"}, data: countries }, invalidMessage: "You need to choose country" },
+						{ view: "combo", label: _("Status"), value: 1, name: "Status", options: { body: {template:"#Name#"}, data: statuses }, invalidMessage: "You need to choose status" },
+						{ 
+							view: "button", 
+							label: _("Save"), 
+							align: "right", 
+							width: 100, 
+							css: "webix_primary", 
+							hotkey:"enter",
+							click: () => this.saveItem()
+						}
+					],
 				},
 				{}
-			]
+			],
+			rules: {
+				Name: webix.rules.isNotEmpty,
+				Email: webix.rules.isNotEmpty && webix.rules.isEmail,
+				Country: webix.rules.isNotEmpty,
+				Status :webix.rules.isNotEmpty
+			}
 		};
 	}
-}
\ No newline at end of file
+
+	saveItem(){
+		const form = this.getRoot();
+		if (form.validate()) {
+			const item_data = form.getValues();
+			if (item_data.id) {
```

Comment:
as I wrote earlier, it is better to separate add/ update logic as you have two buttons (save and add) in the UI

Reply:
Added new logic to “Add” button.